src/plotting.jl

```diff
@@ -47,6 +64,8 @@ function _plot(
     width::Real = 1200,
     height::Real = 600,
     plot_directory::AbstractString = default_plot_directory(),
+    plot_filename::AbstractString = default_plot_filename(),
+    file_extensions = ["svg", "png"],
```

Comment:
I like svg more for the vector graphics, and you can add them to docs. But you can't share them online, hence the pngs. So I think these are good defaults.